src/converters/toZigbee.ts

```diff
@@ -4315,8 +4315,18 @@ const converters2 = {
             );
 
             if (isGroup || (utils.isObject(removeresp) && (removeresp.status === 0 || removeresp.status == 133 || removeresp.status == 139))) {
+                let commandTransitionTime;
```

Comment:
This can be simplified to:

```js
const addSceneCommand. = Number.isInteger(transtime) ? 'add' : 'enhancedAdd';
const commandTransitionTime = addSceneCommand === 'enhancedAdd' ? Math.floor(transtime * 10) : transtime
```